deliver/lib/deliver/upload_metadata.rb

```diff
@@ -89,7 +89,7 @@ def upload(options)
       enabled_languages = detect_languages(options)
 
       app_store_version_localizations = verify_available_version_languages!(options, app, enabled_languages) unless options[:edit_live]
-      app_info_localizations = verify_available_info_languages!(options, app, enabled_languages) unless options[:edit_live]
+      app_info_localizations = verify_available_info_languages!(options, app, enabled_languages) unless options[:edit_live] || !updating_localised_app_info?(options, app)
```

Comment:
Now this code will first fetch `edit || live`, and then if there are no changes to app info localizations, good. But if there are, it will call `verify_available_info_languages` and will fetch `edit` app info again, which's not necessary since we just fetched it. Do you think this is something we could easily improve? E.g. by fetching them once and passing them as parameters where needed :)

Sometimes extra fetches can significantly hit fastlane's performance, so we try to keep the requests as lean as possible 🙏

Reply:
Thanks. Makes sense. 
I moved the call out of the functions and passed it as a parameter. There was already another call at that level to `fetch_app_info` that felt redundant to me at this point. 
Please, let me know if you see any reason to refetch at that point. I didn't.